activation/nipost.go

```diff
@@ -206,11 +209,13 @@ func (nb *NIPostBuilder) BuildNIPost(ctx context.Context, challenge *types.NIPos
 	if nb.state.NIPost.Post == nil {
 		nb.log.With().Info("starting post execution", log.Binary("challenge", nb.state.PoetProofRef[:]))
 		startTime := time.Now()
+		events.EmitPostStart(nb.state.PoetProofRef[:])
```

Comment:
Any reason why this event (and the one below) take `[]byte` as parameter instead of the actual type?

Reply:
tried to change that, but zero challenge is defined as a slice, so i would rather leave it as it is